src/sentry/runner/initializer.py

```diff
@@ -67,6 +67,7 @@ def get_asset_version(settings):
     'system.databases': 'DATABASES',
     'system.debug': 'DEBUG',
     'system.secret-key': 'SECRET_KEY',
+    'system.rate-limit': 'SENTRY_SYSTEM_MAX_EVENTS_PER_MINUTE',
```

Comment:
To be clear, this mapping is here is purely for things that Django would rely on and some other edge cases in backwards compatibility that don't support an easy change to using options values.